src/content/en/updates/2019/05/web-share-files.md

```diff
@@ -0,0 +1,103 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chrome 75 introduces file sharing from a web app, which lets your web app can share with virtually anything on a user's device.
+
+{# wf_published_on: 2019-05-02 #}
+{# wf_updated_on: 2019-05-02 #}
+{# wf_featured_image: /web/updates/images/generic/share.png #}
+{# wf_tags: capabilities,sharing,chrome75 #}
+{# wf_featured_snippet: Chrome 75 introduces file sharing from a web app, which lets your web app can share with virtually anything on a user's device. #}
+{# wf_blink_components: Blink>WebShare #}
+
+# Web share supports files {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+<figure class="attempt-right">
+  <img src="/web/updates/images/2018/12/wst-send.png">
+  <figcaption>
+    System-level share target picker with installed PWA as an option.
+  </figcaption>
+</figure>
+
+Over the last few years, we've been working to bring native sharing capabilities
+to the web. The
+[Web Share API](/web/updates/2016/09/navigator-share)
+allows web apps to invoke the same share dialog box that a native app user would
+see. The
+[Web Share Target API](/web/updates/2018/12/web-share-target)
+allows web apps to receive data from a share.
+
+The only resource previously supported by these APIs was files. Chrome 75
+introduces file sharing from a web app with its implementation of the [Web Share
```

Comment:
Can you make this more clear, maybe "Chrome 75 adds support for Web Share API Level 2, making it easy for web apps to share files to other apps using the system provided picker."